resources/definitions/tenlog_printer.def.json

```diff
@@ -0,0 +1,295 @@
+{
+    "name": "tenlog_printer",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": { 
+	"visible": false,
+        "author": "Tenlog",
+        "manufacturer": "Tenlog",
+        "weight": 4,
+
+        "has_variants": false,
+        "variants_name": "Nozzle Size",
+        "preferred_variant_name": "0.4mm Nozzle",
+
+	"file_formats": "text/x-gcode",
+        "has_materials": true,
+	"first_start_actions": ["MachineSettingsAction"],
+
+        "has_machine_quality": false,
+        "preferred_quality_type": "draft",
+        "preferred_material": "tlgreen_pla",
```

Comment:
There is no material `tlgreen_pla`.

Reply:
This preferred material doesn't exist. What is the material that should get loaded in automatically with the printer? Perhaps you intended to add these material profiles to https://github.com/Ultimaker/fdm_materials/ as well? Or else just select `generic_pla_175`?